.travis.yml

```
@@ -10,10 +10,9 @@ cache:
 matrix:
   allow_failures:
       - rust: nightly
-before_script:
-  - (test -x $HOME/.cargo/bin/cargo-install-update || cargo install cargo-update)
-  - (test -x $HOME/.cargo/bin/mdbook || cargo install --vers "^0.1" mdbook)
-  - cargo install-update -a
+before_install:
+  - cargo install mdbook --vers '0.3.1' --debug --force
```

```suggestion
  - cargo install mdbook --version '0.3.1' --debug --force
```

(`--vers` seems to be some old back-compat thing, `-h` tells to use `--version`)